doc/lib.md

```diff
@@ -480,9 +558,12 @@ Miscellaneous
 * [with](with.html)
   This module implements the `with` macro for easy function chaining.
 
+* [wrapnils](wrapnils.html)
+  This module allows evaluating expressions safely against nil dereferences and more.,
```

Comment:
```suggestion
  This module allows evaluating expressions safely against nil dereferences.
```

Reply:
The module also mentions covering for:

> field accesses with incorrect discriminant in case objects

which was too long to include in this description. But I guess it's fine to elude.